examples/graphql/docker/docker-compose.yaml

```diff
@@ -10,7 +10,7 @@ services:
     ports:
       - "9464:9464"
       - "4317:4317"
-      - "55681:55681"
+      - "4318:4318"
```

Comment:
The image in use (line 6) is `otel/opentelemetry-collector:0.27.0` which, to my understanding, is not exposing the 4318 port yet (only the legacy 55681 port).
I think we should apply the port change along with an upgrade to the collector image version which includes the new endpoint.